lib/src/feedback_widget.dart

```diff
@@ -169,27 +172,39 @@ class FeedbackWidgetState extends State<FeedbackWidget>
                   },
                 ),
               ),
-              if (widget.isFeedbackVisible)
+              // only display if feedback is visible or this widget is still
+              // animating out
+              if (widget.isFeedbackVisible || !animation.isDismissed)
                 Positioned(
                   key: const Key('feedback_bottom_sheet'),
                   left: 0,
                   // Make sure the input field is always visible,
                   // especially if the keyboard is shown
                   bottom: MediaQuery.of(context).viewInsets.bottom,
                   right: 0,
-                  child: SizedBox(
-                    height: MediaQuery.of(context).size.height,
-                    child: FeedbackBottomSheet(
-                      onSubmit: (context, feedback) async {
-                        await _sendFeedback(
-                          context,
-                          FeedbackData.of(context)!.onFeedback!,
-                          screenshotController,
-                          feedback,
-                          widget.pixelRatio,
-                        );
-                        painterController.clear();
-                      },
+                  height: MediaQuery.of(context).size.height *
+                      // height should be screen size minus the bottom edge of
+                      // screenshot widget:
+                      //   1 - (scaleOrigin + height*scaleFactor)
+                      (1 - (.35 / 2 + 1.65 / 2 * .65)),
```

Comment:
This one's a doozy, but it ensures that the bottom sheet is programmatically scaled relative to the screenshot.